R/mrmash_wrapper.R

```diff
@@ -291,30 +273,66 @@ compute_w0 <- function(Bhat, ncomps) {
   return(w0)
 }
 
+
 ### Filter data-driven matrices
-filter_data_driven_mats <- function(Y, data_driven_mats) {
+filter_data_driven_mats <- function(Y, data_driven_mats, data_driven_mats_w = NULL, prior_weights_min = 1e-4) {
```

Comment:
Again it does not seem to make sense to me to have two functions `filter_data_driven_mats` and `filter_data_driven_priors` . Please carefully merge them.

Reply:
This is the function meant to be used and actually used in the multivariate pipeline.